src/Runtime.php

```diff
@@ -69,17 +76,18 @@ public function getImageFigure(
         string $figureDataAttributes = null,
         string $imgDataAttributes = null
     ): string {
-        $path = $this->processPath($path);
-        $nonLazyLoadImgMarkup = $this->getNonLazyLoadImgMarkup(
+        $path = $this->processPath($path, $srcFilter);
+        $nonLazyLoadImgMarkup = $this->getImageImg(
             $path,
             $srcFilter,
             $srcsetFilters,
             $alt,
             $imgClass,
             $sizes,
             $imgImportance,
-            $imgDataAttributes
+            $imgDataAttributes,
```

Comment:
I’m not sure finishing args list with a coma is working before PHP 8.0

Reply:
Fixed